tests/neuro_san/internals/chat/test_connectivity_reporter.py

```diff
@@ -50,8 +50,9 @@ def test_hello_world(self):
         """
         agent_network: AgentNetwork = self.get_sample_registry("hello_world.hocon")
         reporter = ConnectivityReporter(agent_network)
+        agent_toolbox_info_file: str = agent_network.get_agent_toolbox_info_file()
 
-        messages: List[Dict[str, Any]] = reporter.report_network_connectivity()
+        messages: List[Dict[str, Any]] = reporter.report_network_connectivity(agent_toolbox_info_file)
```

Comment:
Given the principle of not changing the main interface (see ContextTypeToolboxFactory comments below),
This file would not have to change at all if the information here were coming into the ToolboxFactory contstructor because the signature for load() would stay intact.